packages/shopify-codemod/test/fixtures/assert/threw-method-to-assert-threw/basic.output.js

```diff
@@ -0,0 +1,2 @@
+assert.threw(foo);
+assert.threw(this.foo.bar);
```

Comment:
No, and now that I look at it, it's horribly broken in other ways :/  I'm probably going to remove this from the PR, because the fix isn't trivial.